src/pages/en/core-concepts/endpoints.md

```diff
@@ -0,0 +1,275 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Endpoints
```

Comment:
I'm totally fine with calling these Endpoints, but just for reference I've also seen them called **API Routes**. I'm assuming the docs team discussed it and decided on Endpoints, but just in case not, there's another name we've used.